src/coreclr/jit/optimizer.cpp

```diff
@@ -9041,12 +9041,742 @@ bool Compiler::optIsRangeCheckRemovable(GenTree* tree)
     return true;
 }
 
-/******************************************************************************
- *
- * Replace x==null with (x|x)==0 if x is a GC-type.
- * This will stress code-gen and the emitter to make sure they support such trees.
- */
+//
+//  optOptimizeBoolsCondBlock:  Optimize boolean when bbJumpKind of both b1 and b2 are BBJ_COND (conditional jump).
+//                              For example, (x==0 && y==0 && z==0) generates
+//                                  b1: GT_JTRUE (BBJ_COND)
+//                                  b2: GT_JTRUE (BBJ_COND)
```

Comment:
Header comment style (see the jit style guide) is:
```suggestion
//------------------------------------------------------------------------
//  optOptimizeBoolsCondBlock:  Optimize boolean when bbJumpKind of both b1 and b2 are BBJ_COND
```
and move the remaining details to a "notes" section after the arguments (and return value, when there is one).

Reply:
Done.